SConstruct

```diff
@@ -676,3 +715,32 @@ vcxproj = base.VSProject(
     VSPROJECT_ROOT_DIRS = ['src/beast', 'src', '.'],
     VSPROJECT_CONFIGS = msvc_configs)
 base.Alias('vcxproj', vcxproj)
+
+#-------------------------------------------------------------------------------
+
+# Adds a phony target to the environment that always builds
+# See: http://www.scons.org/wiki/PhonyTargets
+def PhonyTargets(env = None, **kw):
```

Comment:
This is a function - it must be called `phony_targets`.  Also, no spaces around `=` _only_ when in a function argument.

Source: https://www.python.org/dev/peps/pep-0008/ - Python is fortunate in having a very specific style guideline as part of the language...